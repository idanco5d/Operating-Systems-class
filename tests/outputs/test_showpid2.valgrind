==29985== Memcheck, a memory error detector
==29985== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==29985== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==29985== Command: /mnt/c/Users/illus/os/hw/1/smash
==29985== Parent PID: 29980
==29985== 
==29985== 
==29985== FILE DESCRIPTORS: 4 open (3 std) at exit.
==29985== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_showpid2.valgrind
==29985==    <inherited from parent>
==29985== 
==29985== 
==29985== HEAP SUMMARY:
==29985==     in use at exit: 0 bytes in 0 blocks
==29985==   total heap usage: 39 allocs, 39 frees, 80,435 bytes allocated
==29985== 
==29985== All heap blocks were freed -- no leaks are possible
==29985== 
==29985== For lists of detected and suppressed errors, rerun with: -s
==29985== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
