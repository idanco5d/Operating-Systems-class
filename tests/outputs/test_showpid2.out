smash> smash pid is 9709
smash> smash pid is 9709
smash> smash pid is 9709
smash> 